imap_processing/mag/constants.py

```diff
@@ -59,6 +59,23 @@ class PrimarySensor(Enum):
     MAGI = 1
 
 
+class VecSec(Enum):
+    """Enum for all valid vector rates (Vectors per second)."""
+
+    ONE_VEC_PER_S = 1
```

Comment:
The enum is already called VecSec, so I think you could shorten things and go `VecSec.ONE` everywhere (maybe make the enum `VecPerSec`?
```suggestion
    ONE = 1
```

Reply:
I will contemplate this one. I think you're right. I might not change it in this PR because I actually have a lot of changes in the next iteration of this code that would make merging very annoying, but perhaps in the next PR I will rework it.